composer.json

```
@@ -13,13 +13,19 @@
     "email": "support@snowplow.io"
   },
   "require": {
-    "php": ">=7.4",
+    "php": ">=8.1",
     "ramsey/uuid": "^3 || ^4"
   },
   "require-dev": {
     "phpunit/phpunit": "^9.5.13",
     "php-coveralls/php-coveralls": "^2.5.2",
-    "phpunit/php-token-stream": "^4.0.4"
+    "phpunit/php-token-stream": "^4.0.4",
+    "symfony/config": "6.4.0",
```

Just one last thing to allow for new patch and minor versions:

```suggestion
    "symfony/config": "^6.4.0",
```